examples/cube/main.rs

```diff
@@ -274,19 +271,27 @@ impl framework::Example for Example {
             }],
         };
 
-        let vs_module = device.create_shader_module(&wgpu::include_spirv!("shader.vert.spv"));
-        let fs_module = device.create_shader_module(&wgpu::include_spirv!("shader.frag.spv"));
+        let mut flags = wgpu::ShaderFlags::VALIDATION;
```

Comment:
nit: we could temporarily have a helper function for this in framework to remove backend matching from the example code

Reply:
I considered this, but the problem is that each example can be limited by the specifics of the shaders it needs. So one can enable experimental translation on Metal, for example, another can not. So for now I think having them separate is better.

Reply:
Yeah for now it's probably fine like this – it might be nice to have a helper once it's ready on all examples